.github/workflows/macOS-tests.yml

```diff
@@ -7,6 +7,8 @@ on:
       - develop
       - hotfix
       - unstable
```

Comment:
Also, please delete `unstable` branch, since as it seems i've forgot to do it myself on release.